contents.json

```diff
@@ -3284,6 +3284,26 @@
         "http://a5.mzstatic.com/us/r30/Purple2/v4/1c/d0/2d/1cd02d13-9d9c-d115-1605-3fb4dae72831/screen696x696.jpeg"
       ]
     },
+    {
+      "title": "GeoLicious",
+      "category-ids": [
+        "location"
+      ],
+      "tags": [
+        "objc"
+      ],
+      "screenshots": [
+        "https://camo.githubusercontent.com/7ac84e87a006d126f5d800002b08da4e3f4e95f5/687474703a2f2f61342e6d7a7374617469632e636f6d2f75732f7233302f507572706c6534322f76342f64632f36352f30642f64633635306463382d323732642d646134612d386563642d3730383236303630363035622f73637265656e363936783639362e6a706567",
+        "https://camo.githubusercontent.com/8cb98bf5ad326ee48ec0a6e50670714e2a411232/687474703a2f2f61352e6d7a7374617469632e636f6d2f75732f7233302f507572706c6536322f76342f66612f30372f33302f66613037333036332d343464392d613733632d313436372d3664613961353464373738662f73637265656e363936783639362e6a706567"
+      ],
+      "description": "Location Tracking made easy... GeoLicious is a long term iOS project I've maintained from 2013 to 2017 and made public in May 2017.",
```

Comment:
@da3x remove "iOS" from the description, that is redundant. As there is everything is an **iOS** app, project, etc :-)

Reply:
...just copied from my original description – why not.